2 - RTB/Documentazione interna/Verbali/Interni/23-11-13.typ

```diff
@@ -0,0 +1,65 @@
+#import "/template.typ": *
+
+#show: project.with(
+  title: "Verbale 13/11/23",
+  docType: "verbale",        
```

Comment:
Manca il subtitle

Reply:
ne abbiamo discusso oggi in meeting, nei verbali abbia deciso che il subtitle non è obbligatorio